packages/account/imAccount/validator/P256Owner.ts

```diff
@@ -76,7 +72,8 @@ export class P256Owner implements WebAuthnValidatorOwner {
       counter: Number(this.defaultSignCount)
     }
     return {
-      signature: signature as HexString,
+      r: r,
+      s: s,
```

Comment:
nit: Maybe wrap them in one signature field (?

```suggestion
      signature: {
         r,
         s,
      }
```

And if the variable name is same as object field name, in JS/TS it can abbreviate the assignment to only the object field name.